README.md

```diff
@@ -138,6 +138,15 @@ If you have any questions or discussion topics, start a [GitHub Discussion][8].
 ## Contributing
 Contributions are always welcome. Please read [Contributing.md](CONTRIBUTING.md) if you are considering making contributions.
 
+## Related projects
+- [i3-resurrect](https://github.com/JonnyHaystack/i3-resurrect) - "Simple solution to saving and restoring i3 workspaces"
+- [tmux-resurrect](https://github.com/tmux-plugins/tmux-resurrect)
```

Comment:
Can this be moved to the Similar Software section in the develop branch? Here's the description that can be used (make sure to update the table of contents as well).
```markdown
### tmux-resurrect
[Tmux-resurrect](https://github.com/tmux-plugins/tmux-resurrect) is a Tmux plugin to restore your tmux environment after a system restart.
````